dashboard/requirements.txt

```diff
@@ -0,0 +1,9 @@
+pylint
+pytest
+streamlit
+altair
+boto3
+python-dotenv
+psycopg2-binary
+st-pages
+pandas
```

Comment:
Do you use pandas yet?

Reply:
Not yet, but based on previous work I expect to use it.